<launch>

    <arg name="leader_id" default="panda_leader" />
    <arg name="follower_id" default="panda_follower" />
    <arg name="gripper_homed" default="false" />

    <!-- <node name="haptic_teleoperation_arm_node" type="haptic_based_teleoperation.py" pkg="panda_teleoperation" output="screen">
    </node> -->

    <node name="teleop_gripper_node" pkg="franka_example_controllers" type="teleop_gripper_node" respawn="false" output="screen" >
        <rosparam param="gripper_homed" subst_value="true">$(arg gripper_homed)</rosparam>
        <remap from="~leader/joint_states" to="$(arg leader_id)/franka_gripper/joint_states" />
        <remap from="leader/homing" to="$(arg leader_id)/franka_gripper/homing" />
        <remap from="follower/homing" to="$(arg follower_id)/franka_gripper/homing" />
        <remap from="follower/grasp" to="$(arg follower_id)/franka_gripper/grasp" />
        <remap from="follower/move" to="$(arg follower_id)/franka_gripper/move" />
        <remap from="follower/stop" to="$(arg follower_id)/franka_gripper/stop" />
    </node>
  
</launch>
